setup.cfg

```diff
@@ -16,6 +18,7 @@ classifiers =
     Programming Language :: Python :: 3.6
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9

```

Comment:
Should be added to test matrix

Reply:
Yes, good point, will add after I get the current code passing tests :)